pkg/storageos/webhook.go

```diff
@@ -0,0 +1,110 @@
+package storageos
+
+import (
+	"github.com/storageos/cluster-operator/pkg/util/k8s"
+	admissionv1 "k8s.io/api/admissionregistration/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	// MutatingWebhookConfigName is the name used for the mutating webhook
+	// configuration.
+	MutatingWebhookConfigName = "storageos-mutating-webhook"
+
+	// MutatingWebhookServiceName name is the name of the service that will be
+	// created for the mutating webhook.
+	MutatingWebhookServiceName = "storageos-mutating-webhook"
```

Comment:
This service name will be common to the mutating webhook and validating webhooks in the future. Let's move this into a common variable with a common name, not specific to any type of webhook.

Reply:
Changed to `storageos-webhook`